examples/coupled_global_model/coupling_interface.jl

```diff
@@ -0,0 +1,119 @@
+# This document lays out the functions that must be extended to
+# use an atmospheric simulation in ClimaOcean.
+
+using ClimaOcean
+
+import Oceananigans: time_step!
+import Oceananigans.Models: update_model_field_time_series!
+
+# Make sure the atmospheric parameters from SpeedyWeather can be used in the compute fluxes function
+import ClimaOcean.OceanSeaIceModels.Atmospheres: thermodynamics_parameters, 
+                                                 boundary_layer_height, 
+                                                 surface_layer_height,
+                                                 regrid_fluxes_to_atmospheric_model!, 
+                                                 interpolate_atmospheric_state!
+
+# Assuming there is a type called ClimaAtmosSimulation encapsulating
+# the atmosphere simulation:
+
+# This can be left blank
+update_model_field_time_series!(::ClimaAtmosSimulation, time) = nothing
+
+# Take one time-step
+time_step!(atmos::ClimaAtmosSimulation) = nothing
+
+# The height of surface variables, used by the turbulent flux solver
+surface_layer_height(s::ClimaAtmosSimulation) = 10 # meters, for example
+
+# This is a parameter that is used in the computation of the fluxes,
+# It probably should not be here but in the similarity theory type.
+boundary_layer_height(atmos::ClimaAtmosSimulation) = 600
+
+# Note: possibly, can use the atmos thermodynamic parameters directly here.
+thermodynamics_parameters(atmos::ClimaAtmosSimulation) = 
+    PrescribedAtmosphereThermodynamicsParameters(Float32)
+
+"""
+    interpolate_atmospheric_state!(surface_atmosphere_state, 
+                                        interpolated_prescribed_freshwater_flux, 
+                                        atmos::ClimaAtmosSimulation, 
+                                        grid, clock)
+
+Interpolate the atmospheric state in `atmos` to `surface_atmospheric_state`, a
+the collection of `Field`s needed to compute turbulent fluxes.
+"""
+function interpolate_atmospheric_state!(surface_atmosphere_state, 
+                                        interpolated_prescribed_freshwater_flux, 
```

Comment:
river runoff and icebergs

Reply:
https://github.com/CliMA/ClimaOcean.jl/blob/main/src/DataWrangling/JRA55.jl

Reply:
https://github.com/CliMA/ClimaOcean.jl/blob/459d76da9ebe6e24d0fc6381b134d140c96bc018/src/DataWrangling/JRA55.jl#L687